Makefile.am

```diff
@@ -137,6 +144,13 @@ if ENABLE_ALL_STATIC
 jq_LDFLAGS += -all-static
 endif
 
+bin_PROGRAMS += src/gen_jq_builtins
+gen_jq_builtins_SOURCES = src/gen_jq_builtins.c
+
+if WIN32
+gen_jq_builtins_LIBADD = -lshlwapi
```

Comment:
Curious, what requires this? dugg around a bit and in https://github.com/jqlang/jq/issues/911 it seems to be needed for some `PathIsRelativeA`

Reply:
*shruggie* I still have not fixed the win32 build =)